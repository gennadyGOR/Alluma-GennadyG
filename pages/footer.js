import BasePage from "./Base.page";

class Footer extends BasePage{

  get footer() { return $(".gg-footer")}

}
export default new Footer();