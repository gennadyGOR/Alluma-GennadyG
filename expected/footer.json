{
  "footer": {
    "titleAbout": "About this site",
    "subTitleLink1": "Site Link 1",
    "subTitleLink2": "Site Link 2",
    "titleHelp": "Get help",
    "subTitlePhoneNumber": "888-888-8888",
    "subTitlePhoneEmail": "Support@email.com",
    "subTitleFax": "Fax: 777-777-7777",
    "titleStayInTouch": "Stay in touch",
    "facebookUrl": "https://www.facebook.com/WeAreAlluma/?ref=page_internal",
    "twitterUrl": "https://twitter.com/WeAreAlluma",
    "linkedinUrl": "https://www.linkedin.com/company/we-are-alluma/",
    "allumaUrl": "https://www.alluma.org/",
    "copyright": "Copyright 2019",
    "allumaLinkText": "Alluma",
    "privacyPolicy": "Privacy Policy",
    "nDiscriminationPolicy": "Non-discrimination Policy",
    "termsOfUse": "Terms of Use"
  }
}